sdk/iot/provisioning/samples/src/paho_iot_provisioning_example.c

```diff
@@ -0,0 +1,304 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <paho-mqtt/MQTTClient.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <az_iot_provisioning_client.h>
+#include <az_result.h>
+#include <az_span.h>
+
+// Service information
+#define GLOBAL_PROVISIONING_ENDPOINT_DEFAULT "ssl://global.azure-devices-provisioning.net:8883"
```

Comment:
is this not a value in our SDK?

Reply:
No, GDE is not the same in all Azure clouds. This is just the one for the Azure public cloud.
Also, this string format is particular to Paho.